src/docs/command-line.dox

```diff
@@ -34,7 +36,10 @@ The \c wt tool supports several commands.  If configured in the underlying
 database, some commands will run recovery when opening the database.  If
 the user wants to force recovery on any command, use the \c -R option.
 In general, commands that modify the database or tables will run recovery
-by default and commands that only read data will not run recovery.
+by default and commands that only read data will not run recovery. It is
+recommended when attempting to diagnose a corrupt database the -r flag
+is supplied. This will prevent utility commands from writing to any of the
```

Comment:
Minor wordsmithing. I suggest:
`It is recommended when attempting to diagnose a corrupt database, that the -r flag be used. This flag will open the connection read-only and prevent utility commands from writing...`